### PR TITLE
Upgrade storyblok rich text rendrer to @storyblok/richtext

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Subj.

- Restore our overloads for rendering links and images
- Remove `storyblok-rich-text-react-renderer` which used old API and did not have scheduled updates

This solves deprecation warnings when rendering rich text

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
* **#4756** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4756?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/.gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .vercel
 # Local cache, used in WebStorem GraphQL plugin
 schema.graphql
+stories.json
```
Comment (alebedev) [inline apps/store/.gitignore:5]: I'm tired of accidentally committing Storyblok dump, this prevents it

**Review by guilhermespopolin — APPROVED** [review]
We probably want Peter testing it a little bit as there's might be some weird things between what CMS send to us. That's why we had to have custom render options before.

**Review by gustaveen — APPROVED** [review]
Good work! 🙌 A bit weird that they don't provide the utils for rendering, a lot of manual conversion compared to the previous package

**Comment by alebedev** [comment]
@gustaveen to be fair, old version used external library for React integration. Storyblok's renderer just returned HTML as a string

**Comment by alebedev** [comment]
### Merge activity

* **Sep 23, 3:21 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4756).

**Pull request merged**